yt_dlp/extractor/voicy.py

```diff
@@ -0,0 +1,145 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..compat import compat_str
+from ..utils import (
+    ExtractorError,
+    smuggle_url,
+    traverse_obj,
+    unsmuggle_url,
+    unified_strdate,
+)
+
+import itertools
+
+
+class VoicyBaseIE(InfoExtractor):
+    # every queries are assumed to be a playlist
+    def _extract_from_playlist_data(self, value):
+        voice_id = compat_str(value['PlaylistId'])
+        upload_date = unified_strdate(value['Published'], False)
+        items = [self._extract_single_article(voice_data) for voice_data in value['VoiceData']]
+        return {
+            '_type': 'playlist',
+            'entries': items,
+            'id': voice_id,
+            'title': compat_str(value['PlaylistName']),
+            'uploader': value['SpeakerName'],
+            'uploader_id': compat_str(value['SpeakerId']),
+            'channel': value['ChannelName'],
+            'channel_id': compat_str(value['ChannelId']),
+            'upload_date': upload_date,
+        }
+
+    # NOTE: "article" in voicy = "track" in CDs = "chapter" in DVDs
+    def _extract_single_article(self, entry):
+        formats = [{
+            'url': entry['VoiceHlsFile'],
+            'format_id': 'hls',
+            'ext': 'm4a',
+            'vcodec': 'none',
+            'protocol': 'm3u8_native',
+        }, {
```

Comment:
Did you miss `acodec`, or is it not known? In the latter case, you may want to either set a `quality` or `field_preference` so that sorting is correct

Reply:
I think it was AAC, but cannot be obtained via API or something else.